### ISSUE TITLE
Can I turn off "automatic url shortening via tinyurl" feature in specific irc channel?

### ISSUE THREAD

**Issue by xnuk** [lambdabot/lambdabot#115]

**Comment by int-e** [comment]
No, lambdabot really lacks some infrastructure for per-channel settings. Maybe you can achieve what you want by editing the URL blacklist (`ignoredStrings`) in the `Lambdabot.Plugin.Reference.Url` module from the lambdabot-reference-plugins package. To actually blacklist a channel, checking the result of `getTarget` in the `contextual` hook seems possible as well.

I'll leave this issue open as a reminder that such configuration should be easier.
